packages/components/MetamaskConnect.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect } from 'react';
+import { Button, Platform, Text, View } from 'react-native';
+
+import { connectWallet, getCurrentWalletConnected } from './util/mfWeb3';
+import { ConnectedWallet } from './util/types';
+
+export const MetamaskConnect: React.FC = () => {
```

Comment:
Was planning on using the "components" package more for UI only stuff. Maybe we should have a "utils" package for all these various helpers and hooks and what not.

I feel like it might be cleaner to extract all the web3 logic in this component into its own hook thats UI agnostic . Similar to what useDapp does: https://github.com/EthWorks/useDApp/tree/master/packages/core/src/hooks